firmware/quadruna/VC/src/app/app_globals.h

```diff
@@ -1,8 +1,29 @@
 #pragma once
 
+#include <stdbool.h>
+#include "app_heartbeatMonitor.h"
+#include "io_led.h"
+#include "io_buzzer.h"
+#include "app_timer.h"
+
+#define BUZZER_ON_DURATION_MS 2000
+
+typedef struct
+{
+    const Buzzer *const buzzer;
+} GlobalsConfig;
+
 typedef struct
 {
-    int unused;
+    const GlobalsConfig *config;
+    bool                 torque_vectoring_switch_is_on;
```

Comment:
do not use this as a global. refetch in the two locations it is used (because rn there is a bug where whatever it reads when it enters drive state gets latched, i.e. if it gets switched off during drive state, it stays on)